platform/security/sts/security-sts-x509validator/src/test/java/org/codice/ddf/security/validator/x509/TestX509PathTokenValidator.java

```diff
@@ -160,4 +160,150 @@ public void testValidateBadToken() {
                 tokenValidatorResponse.getToken()
                         .getState());
     }
+
+    @Test
+    public void testAdditionalPropertyEmail() {
+        try {
+            Credential credential = mock(Credential.class);
+            X509Certificate x509Certificate = mock(X509Certificate.class);
+            X500Principal x500Principal = new X500Principal(
+                    "cn=myxman,ou=someunit,o=someorg,EMAILADDRESS=name@example.com");
+            when(x509Certificate.getSubjectX500Principal()).thenReturn(x500Principal);
+            X509Certificate[] x509Certificates = new X509Certificate[] {x509Certificate};
+            when(credential.getCertificates()).thenReturn(x509Certificates);
+            when(validator.validate(any(Credential.class), any(RequestData.class))).thenReturn(
+                    credential);
+        } catch (WSSecurityException e) {
+            //ignore
+        }
+        x509PathTokenValidator.setValidator(validator);
+
+        TokenValidatorParameters tokenParameters = mock(TokenValidatorParameters.class);
+        STSPropertiesMBean stsPropertiesMBean = mock(STSPropertiesMBean.class);
+        when(tokenParameters.getStsProperties()).thenReturn(stsPropertiesMBean);
+        Crypto crypto = mock(Crypto.class);
+        when(stsPropertiesMBean.getSignatureCrypto()).thenReturn(crypto);
+        ReceivedToken receivedToken = mock(ReceivedToken.class);
+        doCallRealMethod().when(receivedToken)
+                .setState(any(ReceivedToken.STATE.class));
+        doCallRealMethod().when(receivedToken)
+                .getState();
+        when(tokenParameters.getToken()).thenReturn(receivedToken);
+        when(receivedToken.isBinarySecurityToken()).thenReturn(true);
+
+        BinarySecurityTokenType binarySecurityTokenType = mock(BinarySecurityTokenType.class);
+        when(binarySecurityTokenType.getValueType()).thenReturn(X509TokenValidator.X509_V3_TYPE);
+
+        when(receivedToken.getToken()).thenReturn(binarySecurityTokenType);
+        when(binarySecurityTokenType.getEncodingType()).thenReturn(X509PathTokenValidator.BASE64_ENCODING);
+        when(binarySecurityTokenType.getValue()).thenReturn("data");
+
+        TokenValidatorResponse tokenValidatorResponse = x509PathTokenValidator.validateToken(
+                tokenParameters);
+
+        assertEquals(ReceivedToken.STATE.VALID,
```

Comment:
According to our Unit Testing guidelines, we should favor `assertThat`over `assertEquals` and the ordering should be `actual, expected`